src/subclass/object.rs

```diff
@@ -65,58 +65,6 @@ pub trait ObjectImpl: ObjectImplExt + 'static {
     }
 }
 
-unsafe extern "C" fn get_property<T: ObjectSubclass>(
```

Comment:
Same for the virtual method ones below, those are nicer outside as they contain quite some code and are not simple trampolines.

Reply:
Especially, for other classes where you have dozens of virtual methods you'd otherwise have a huge block of functions inline inside the `class_init` function. Here we only have three and it's quite hard to read already if not handled as separate functions like it was before.